src/AppBundle/Resources/views/Title/show.json.twig

```diff
@@ -0,0 +1,43 @@
+
+{
+    "@context": "http://schema.org",
+    "@type": "Book",
+    
+    "url": "{{url('title_show', {'id': title.id}) }}",
+    
+    "name": "{{ title.title }}",
+    
+    {% if title.source and title.source.name != 'unknown' and title.source.name != 'None' %}  
+    "source": "{{ title.source.name }}",
+    "identifier": "{{ title.sourceId }}",
+    {% endif %}
+    
+    {% if title.source2 and title.source2.name != 'unknown' and title.source2.name != 'None' %}
+    "source": "{{ title.source2.name }}",
+    "identifier": "{{ title.source2Id }}",
+    {% endif %}
+    
```

Comment:
The duplicate keys are a problem here as well. They need to be in a list. To see the problem in action, look at title 2 as json. The json validators won't see one of the sources.